velox/exec/GroupingSet.cpp

```diff
@@ -180,16 +180,19 @@ void GroupingSet::initializeGlobalAggregation() {
 
   // Row layout is:
   //  - null flags - one bit per aggregate,
+  // int32_t row size,
   //  - fixed-width accumulators - one per aggregate
-  int32_t offset = bits::nbytes(aggregates_.size());
+  int32_t rowSizeOffset = bits::nbytes(aggregates_.size());
+  int32_t offset = rowSizeOffset + sizeof(int32_t);
```

Comment:
The PR description says "This is only added if there are variable lenght fields or accumulators.", but this code suggests that rowSize field is always present.

Also, typo in PR description: lenght -> length